docker/Jenkinsfile-build-k8s-test-docker

```diff
@@ -0,0 +1,110 @@
+#! /usr/bin/groovy
```

Comment:
why two jenkins jobs? could have just one with multiple stages?

Reply:
To be able to build each one separately **easily** - tried to make it as obvious as possible there are two images built. I guess **one version** is enough as you say, as well as one script with multiple stages run at once.

Having one version has one more upside : If the versions used do not match, it's obvious there could be a problem.

In reality, those images are often independent.